src/accidental.ts

```diff
@@ -529,16 +550,28 @@ export class Accidental extends Modifier {
     L('Rendering: ', type, accX, accY);
 
     if (!cautionary) {
+      if (!glyph) {
+        throw new Error('glyph is undefined');
+      }
       glyph.render(context, accX, accY);
     } else {
       // Render the accidental in parentheses.
+      if (!parenRight) {
+        throw new Error('parentRight is undefined');
+      }
       parenRight.render(context, accX, accY);
       accX -= getGlyphWidth(parenRight);
       accX -= parenRightPadding;
       accX -= this.accidental.parenRightPaddingAdjustment;
+      if (!glyph) {
+        throw new Error('glyph is undefined');
+      }
       glyph.render(context, accX, accY);
       accX -= getGlyphWidth(glyph);
       accX -= parenLeftPadding;
+      if (!parenLeft) {
+        throw new Error('parenLeft is undefined');
```

Comment:
We are using `throw new Vex.RERR`

Reply:
Perhaps it would make sense to check for parents and `Glyph` once at the beginning of the function.